envelopespec/envelope.go

```diff
@@ -0,0 +1,87 @@
+package envelopespec
+
+import (
+	"errors"
+	"fmt"
+)
+
+// Validate returns an error if env is not well-formed.
```

Comment:
I am hesitant to remove this section of doc comments just to replace it with `Validate returns an error if m contains invalid fields.`. The existing doc comments seem to cover all the peculiarities fo the envelope validation.

Reply:
Yep, it can stay as is.